prettier.config.d.ts

```diff
@@ -0,0 +1,13 @@
+export declare const bracketSpacing: boolean;
```

Comment:
Copyright message needed? (I don't think so)

Reply:
this is auto generated - a glitch in the build system I couldn't figure out